src/components/ProductsListContainer.js

```diff
@@ -0,0 +1,40 @@
+import productsData from '../mocks/en-us/featured-products.json';
```

Comment:
I recommend you to create a hook to obtain this mock data, well all mock data and not import it directly